open-craft/frontend-app-course-authoring
# [FC-0036] feat: tags drawer style refinements

## Description

This PR makes styling updates to the tags drawer and fixes the issue with scrolling.

<img width="680" alt="Screenshot 2024-04-09 at 3 53 30 PM" src="https://github.com/open-craft/frontend-app-course-authoring/assets/6829768/86636b87-10e6-416d-a7d9-7c7b0c4ae798">

## Supporting Information

Related PR:
- Depends on https://github.com/openedx/frontend-app-course-authoring/pull/939

## Testing Instruction

1. Start your local devstack with this PR
1. Make sure you have some sample taxonomy/tags data setup: https://github.com/open-craft/taxonomy-sample-data
1. Go into a course and click on "Manage tags" option for any unit/block
1. Confirm the following:  
    1. There is no "x" in the top right of the drawer
    1. Styling of headings has changed to match [figma](https://www.figma.com/file/DmUC2e7BAoQsGDq4KgApPa/Content-Tagging-MVP?type=design&node-id=11845-19907&mode=design)
    1. Dropdown arrows moved to the left of taxonomy name
    1. Styling has changed for the count of tags per taxonomy (also matching figma)
    1. The tag count shows as "0" for taxonomies without tags, and is greyed out.
    1. Make sure the drawer content is scrollable
---
Private-ref: [FAL-3645](https://tasks.opencraft.com/browse/FAL-3645)
